example/src/App.tsx

```diff
@@ -56,6 +54,7 @@ export default function App() {
       <Camera
         device={device}
         isActive={true}
+        torch="on"
```

Comment:
just wondering why are we turning on the torch at all times here? should we move this into a button which users can toggle?

sometimes this feature actually worsens the scanning rather than improving